src/jquery.ime.js

```diff
@@ -282,7 +285,7 @@
 
 			if ( this.context.length > this.inputmethod.contextLength ) {
 				// The buffer is longer than needed, truncate it at the front
-				this.context = this.context.substring(
+				this.context = this.context.slice(
```

Comment:
Automatic change by eslint, which requires slice() instead of substring() and substr().